pkg/server/handler.go

```diff
@@ -30,6 +30,10 @@ func WrapHandler(handler HandlerFunc) gin.HandlerFunc {
 
 			if gin.Mode() == "debug" {
 				logrus.Debugf("error: %+v", err)
+			} else {
+				// Remove detailed info when in the release mode
+				serverErr.Op = ""
+				serverErr.Err = nil
```

Comment:
Currently login failed is also a kind of error. It doesn't make sense to remove such error information. Internal errors is fine not to return to client